.pipelines/v2/templates/job-test-project.yml

```diff
@@ -34,6 +37,30 @@ jobs:
     fetchDepth: 1
     fetchTags: false
 
+  - powershell: |
+      $edge_url = 'https://go.microsoft.com/fwlink/?linkid=2084649&Channel=Canary&language=en'
+      $timeout = New-TimeSpan -Minutes 6
+      $timeoutSeconds = [int]$timeout.TotalSeconds
+      $command = {
+          Invoke-WebRequest -Uri $using:edge_url -OutFile $(Pipeline.Workspace)\MicrosoftEdgeSetup.exe
+          Write-Host "##[command]Installing Canary channel of Microsoft Edge"
+          Start-Process $(Pipeline.Workspace)\MicrosoftEdgeSetup.exe -ArgumentList '/silent /install' -Wait
+      }
+
+      $job = Start-Job -ScriptBlock $command
+      Wait-Job $job -Timeout $timeoutSeconds
+      if ($job.State -eq "Running") {
+          Stop-Job $job
+          Write-Host "##[warning]The job was stopped because it exceeded the time limit."
+      }
+    displayName: "Install the latest MSEdge Canary"
+    condition: eq('${{ parameters.usingLatestWebView2 }}', true)
+
+  - script: 
+      reg add "HKLM\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannels" /v PowerToys.exe /t REG_SZ /d "0,3"
```

Comment:
Can you combine this with the previous script? It's just one more command after all :)